src/components/Navigation/SharedStyles.js

```diff
@@ -14,60 +14,44 @@ export const H2 = styled(BasicH2)`
   font-weight: ${FONT_WEIGHT.normal};
   color: ${({ theme }) => theme.medium};
 `;
+
+const BACKDROP_COLOR = "rgba(250, 250, 250, 1)";
+const BACKDROP_FADE = "rgba(250, 250, 250, 0)";
+const SHADOW_COLOR = "rgba(0, 0, 0, 0.05)";
+const SHADOW_FADE = "rgba(0, 0, 0, 0)";
+const SHADOW_SIZE = "1.5rem";
+
 export const NavAbsoluteEl = styled.div`
-  position: absolute;
-  overflow: hidden;
-  height: calc(100% - 8.31rem);
-  width: 100%;
-  top: 4rem;
-  bottom: 4.31rem;
+  overflow-y: scroll;
+  flex-grow: 1;
 
-  /* <SideNav/> does not have mobile friendly design (API REF) */
-  @media (${MEDIA_QUERIES.ltLaptop}) {
-    overflow-y: scroll;
+  // Suppress scrollbar on nav
+  @media (${MEDIA_QUERIES.canHover}) {
+    overflow: hidden;
+    &:hover {
+      overflow-y: scroll;
+    }
   }
-
-  &:hover {
+  // <SideNav/> does not have mobile friendly design (API REF)
+  @media (${MEDIA_QUERIES.ltLaptop}) {
     overflow-y: scroll;
   }
 
-  &::before {
-    content: "";
-    z-index: 2;
-    position: fixed;
-    top: 2.68rem;
-    left: 2.5rem;
-    width: 15.62rem;
-    height: 3.125rem;
-    background: rgb(250, 250, 250);
-    background: linear-gradient(
-      180deg,
-      rgba(250, 250, 250, 1) 0%,
-      rgba(250, 250, 250, 0.75) 25%,
-      rgba(250, 250, 250, 0.5) 50%,
-      rgba(250, 250, 250, 0.25) 75%,
-      rgba(250, 250, 250, 0) 100%
-    );
-  }
+  // Edge gradients for scrolling. This is a little more robust than fixed
+  // positioning, which gets weird when the element is offscreen (on mobile or
+  // narrow laptops). See http://lea.verou.me/2012/04/background-attachment-local/
 
-  &::after {
-    content: "";
-    z-index: 2;
-    position: fixed;
-    bottom: 4.312rem;
-    left: 2.5rem;
-    width: 15.625rem;
-    height: 3.125rem;
-    background: rgb(250, 250, 250);
-    background: linear-gradient(
-      0deg,
-      rgba(250, 250, 250, 1) 0%,
-      rgba(250, 250, 250, 0.75) 25%,
-      rgba(250, 250, 250, 0.5) 50%,
-      rgba(250, 250, 250, 0.25) 75%,
-      rgba(250, 250, 250, 0) 100%
-    );
-  }
+  // Shadow covers
+  background: linear-gradient(${BACKDROP_COLOR} 30%, ${BACKDROP_FADE}),
+    linear-gradient(${BACKDROP_FADE}, ${BACKDROP_COLOR} 70%) 0 100%,
+    // Shadows
+      linear-gradient(${SHADOW_COLOR}, ${SHADOW_FADE} 100%),
+    linear-gradient(${SHADOW_FADE}, ${SHADOW_COLOR}) 0 100%;
+  background-repeat: no-repeat;
+  background-size: 100% 3rem, 100% 3rem, 100% ${SHADOW_SIZE},
+    100% ${SHADOW_SIZE};
+  // Opera doesn't support this in the shorthand
+  background-attachment: local, local, scroll, scroll;
```

Comment:
New scroll shadow method, stolen from the linked blog post above.

Reply:
Let's confirm the design of this with Charles

Reply:
Instead of this, I'm gonna change it to hide the fade on small screens. Will circle back in my pt 2 PR.